### PR TITLE
added conf-git.1.0

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by UnixJunkie** [comment]
Tests say it works everywhere (what a happy surprise, for once).
I used the conf-which package as a template, mostly.

File: packages/conf-git/conf-git.1.0/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
+opam-version: "2.0"
+maintainer: "unixjunkie@sdf.org"
+homepage: "https://git-scm.com"
+authors: "Linus Torvalds"
+bug-reports: "https://github.com/ocaml/opam-repository/issues"
+dev-repo: "git+https://github.com/ocaml/opam-repository.git"
+license: "GPL-2+"
+build: [["which" "git"]]
```
Comment (mseri) [inline packages/conf-git/conf-git.1.0/opam]: Does this mean that it depends on `conf-which`?

Comment (kit-ty-kate) [reply packages/conf-git/conf-git.1.0/opam]: yes.

Comment (kit-ty-kate) [reply packages/conf-git/conf-git.1.0/opam]: fixed.

**Comment by UnixJunkie** [comment]
Yes, it depends on conf-which.
Nice catch and thanks for the fix.

**Pull request merged**